config/cable.yml

```diff
@@ -0,0 +1,9 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+production:
+  adapter: redis
+  url: redis://localhost:6379/1
```

Comment:
ActionCable got merged into Rails proper for 5.0, and seems to allow WebSocket connections, so if we want to set up to have some realtime notification system we mmmight use this? (Default config.)

Reply:
What is ActionCable and do we actually use it?

Reply:
It's apparently a thing to do with WebSockets (see [here](https://github.com/rails/rails/tree/master/actioncable)), and not as far as I know, but this is part of the default config and I assume we'd have to do something complicated to remove actioncable as a gem and make this config no longer necessary.